src/device-interface.d.ts

```diff
@@ -24,16 +24,23 @@ interface IdentityObject {
      emailAddress?: string,
 }
 
+interface InternalIdentityObject extends IdentityObject {
+     fullName?: string
+}
```

Comment:
Internal versions of these objects include other properties that we use for convenience. I've created the distinction so we're sure we're sending the right stuff over the API.